src/Jira/Api/Result.php

```diff
@@ -78,6 +78,17 @@ public function getIssues()
 
         return array();
     }
+    
+    public function getErrors()
+    {
+    	if (isset($this->result['errors'])) {
+            $result = array();
+            foreach ($this->result['errors'] as $issue) {
+                $result[] = new Issue($issue);
```

Comment:
Why you need to create `Issue` object with error content? Each entry in `errors` array doesn't represent an issue at all.